addons/point_of_sale/static/src/app/components/popups/preset_slots_popup/preset_slots_popup.xml

```diff
@@ -18,20 +18,20 @@
                     t-attf-class="{{ preset.id === state.selectedPresetId ? '' : 'd-none' }}">
                     <div class="d-flex w-100 flex-wrap gap-2 mt-2">
                         <button class="btn flex-grow-1"
-                            t-foreach="Object.keys(preset.availabilities)" t-as="sqlDate" t-key="sqlDate"
-                            t-attf-class="{{ state.selectedDate === sqlDate ? 'btn-primary' : 'btn-secondary' }}"
-                            t-att-class="{'o_colorlist_item_color_transparent_6': Object.values(preset.availabilities[sqlDate]).length === 0}"
-                            t-on-click="() => state.selectedDate = sqlDate"
-                            t-esc="formatDate(sqlDate)" />
```

Comment:
e.g. here, this is correct. only the `t-esc` (shown to the user) should be formatted.